src/examples/table/TableVerticalScrolling.tsx

```diff
@@ -18,15 +18,17 @@ const rowData = Array.from(Array(10)).map((row, index) => ({
   soil: 'Well draining'
 }));
 
-const StyledHeaderCell = styled(HeaderCell)`
-  box-shadow: inset 0 -1px 0 ${props => getColorV8('neutralHue', 300, props.theme)};
```

Comment:
Removed since v9 `StyledHead` component already [sets](https://github.com/zendeskgarden/react-components/blob/next/packages/tables/src/styled/StyledHead.ts#L26) the `box-shadow` correctly.